### DISCUSSION TITLE
제페토 유니티와, 실행화면이 다릅니다

### DISCUSSION THREAD

**Discussion by hjs6082** [naverz/zepeto-studio-kor#1080] [5. Q&A]
![image](https://user-images.githubusercontent.com/70927848/181155477-e3d408c0-ddbf-4099-87a4-4d6c29bdb16f.png)
제페토 카메라를 사용하지않고, 제페토플레이어스에도 카메라를 메인카메라로 바꿔두고 제페토 카메라를 enabled = false로 해둔 상태입니다. 그냥 캔버스를 만들고, Camera에 main camera를 넣어 준 후에 실행을 하였는데. 메테리얼 색이 다빠지고, 캔버스에 붙어있는 오브젝트들이 사라졌습니다. 이문제 어떻게해결해야할까요

![image](https://user-images.githubusercontent.com/70927848/181155666-9f458c61-1a46-4ba9-881a-a26b7c68b8f4.png)
실행 화면입니다.

**Comment by chlguswls32** [comment]
hjs6082님 안녕하세요.

아래이미지에 가운데부분에 직사각형의 게임오브젝트가있고, 아래이미지 가운데 부분 위쪽에 직사각형의 게임오브젝트가 색이 ADD된 느낌으로 1개더 보이는걸로 봐서, 아래이미지는 카메라 2~3개가 만들어내고있는것 같습니다.

Material의 색이 빠져 보이는 이유는 아마도, UI 카메라가 할당된 구조에보면 UI Canvas부분이 조금 불투명하고 어둡게 씬 전체를 덮고있어보이네요.! ㅎㅎ 

문제해결에 도움이 되었으면 좋겠네요..! ㅎㅎ
